dali/base/dasess.cpp

```diff
@@ -453,6 +453,47 @@ class CEnableScopeScansReq : implements IMessageWrapper
     }
 };
 
+
+//Helper class to Serialize/deserialize digital signature of "key;user;timestamp" and request timestamp
+class CSignatureSerializationHelper
+{
+public:
+    static void serializeSignature(const char * key, IUserDescriptor *udesc, MemoryBuffer &mb)
```

Comment:
I wish these names were a little more self documenting.  This class name makes me think I could use this class for signatures throughout the code.

Is this really CDaliMessageSignatureHelper, or CDfsScopeSignatureHelper?

How useful / generic is this class?

Reply:
Yes a more self documenting name would be great.  Also, for example calling "key", "scope" if indeed it is always a scope.  Then I would immediately know when reading the code what was really happening.